src/main/java/com/kobylynskyi/graphql/codegen/model/MappingConfig.java

```diff
@@ -606,6 +608,15 @@ public Set<String> getFieldsToExcludeFromGeneration() {
         return fieldsToExcludeFromGeneration;
     }
 
+    public void setFieldsWithDataFetcherResult(Set<String> fieldsWithDataFetcherResult) {
```

Comment:
Please fix the order of getter/setter. Should not interfere with other pairs.

Reply:
@kobylynskyi done :)